### PR TITLE
[WIP]Gas Settings: Add in owner page

### PR DESCRIPTION
**Pull request by SaptakS**
##### Description
Gas settings for changing ownership.

##### Checklist

- [ ] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [ ] Linter status: 100% pass
- [ ] Added relevant tests
- [ ] Tests pass and code coverage has not decreased
- [ ] Changes don't break existing behavior
- [ ] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [ ] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [ ] Completed: Code review by core team and any requested changes

###### Contributor

- [ ] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Tested all changes **locally**
- [ ] Verified existing functionality
- [ ] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging

###### Funder

- [ ] Validated requested changes were made to specification
- [ ] Bounty payout released to the contributor

### CODE REVIEW

File: app/assets/v2/js/grants/detail.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ $(document).ready(function() {
         contract_owner_address
       ).send({
         from: accounts[0],
-        gasPrice: 8000000000
+        gasPrice: web3.utils.toHex($('.gas-section-owner [name=gasPrice]').val() * Math.pow(10, 9))
```
Comment (thelostone-mc) [inline app/assets/v2/js/grants/detail.js]: lol why was this even

**Comment by thelostone-mc** [comment]
@SaptakS could you open this against `grantz-cleanup` ?

**Comment by thelostone-mc** [comment]
seem to keep getting this in `/new` 🤔 
screen shot 2018-12-31 at 10 05 02 pm

**Comment by SaptakS** [comment]
@thelostone-mc yes!

**Comment by thelostone-mc** [comment]
Hmm the issue still seems to persist across gas widgets 🤔 
If it's working fine on bounties & grants -> should we throw it up on staging ? @SaptakS

**Comment by SaptakS** [comment]
Yeah. We can put it on staging and test I think.

**Comment by thelostone-mc** [comment]
Yo @SaptakS removed out the change ownership feature for grants :P
So closing this PR out :P

Oh and give this a listen https://open.spotify.com/track/6Q1p0mZ66xLI5WTVUgpukp?si=J7xvsNsmQE6LQdzTMDATVw

**Pull request closed without merging**